x-pack/plugins/rule_registry/server/utils/get_rule_executor_data.ts

```diff
@@ -37,3 +38,14 @@ export function getRuleExecutorData(
     [PRODUCER]: type.producer,
   };
 }
+
+export function getRuleData(options: AlertExecutorOptions<any, any, any, any, any>) {
```

Comment:
do we still need the other function based on the rule type definition? (`getRuleExecutorData`)

Reply:
Good point, I forgot to remove it :+1: